man/GGIR.Rd

```diff
@@ -251,7 +251,20 @@ GGIR(mode = \Sexpr{format(formals(GGIR::GGIR)[["mode"]])},
         For the Actiwatch data functionality has been tested with example data from
         only one research group. Please let us know if you have difficulties using this
         functionality in your data.
-        }
+      }
+       \item{maxRecordingInterval}{
+        Numeric (default = \Sexpr{GGIR::load_params()[["params_general"]][["maxRecordingInterval"]]}).
```

Comment:
When the default setting is NULL, then we need the `format` function wrapping the \Sexpr, otherwise the documentation would show `Numeric (default = ). to indicate...`. It would be:
```
Numeric (default = \Sexpr{format(GGIR::load_params()[["params_general"]][["maxRecordingInterval"]])}).
```

This issue is also in other arguments in the GGIR documentation file, I can revise this in a separate branch if you prefer that.